Demos/Blazorise.Demo/Pages/Tests/ButtonsPage.razor

```diff
@@ -226,21 +226,21 @@
                 <CardTitle>Dropdown Nesting</CardTitle>
             </CardHeader>
             <CardBody>
-                <Dropdown>
+                <Dropdown Direction="Direction.End" Display="Display.InlineBlock">
                     <DropdownToggle Color="Color.Primary">Level 1</DropdownToggle>
                     <DropdownMenu>
                         <DropdownItem>Item 1.1</DropdownItem>
-                        <Dropdown>
+                        <Dropdown Direction="Direction.End">
```

Comment:
I think submenus to the right(End) should be the default option.

Reply:
How so? If nested menus default is right, but if main menu default is down?

Reply:
I think we had diferent CSS selectors for submenus.

Reply:
What do you mean? Since we only did a single direction before for nested menus we had custom css to just make everything go right.

But we can make the default right, the solution will be to apply the dropright classes when direction is default. This is preferable as classless drop down now corresponds to down direction basically, so that should stay as it is now.